setup/gcp/installer/util/gcloud.py

```diff
@@ -431,18 +431,23 @@ def choose_project():
     return target_id
 
 
-def create_or_reuse_service_acct(acct_type, acct_id, advanced_mode, dry_run):
+def create_or_reuse_service_acct(acct_type,
```

Comment:
Can you please tweak each of these names as below?  Actually the id is really the email, right?  Please update if that's the case.  Because it will make this code easier to understand `acct_id[:acct_id.index('@')]`
`service_acct_type`
`service_acct_name`
`service_acct_id` or `service_acct_email`

Reply:
Updated acct_it to acct_email. Other argument names for function create_or_reuse_service_account should be self explanatory based on the function context. Since we are creating/reusing service account so arguments like acct_type/acct_name should reference to service account even without the `service` word in the variable.